src/data/faq_de.json

```diff
@@ -31,14 +31,19 @@
                         ]
                     },
                     {
-                        "title": "So erreichen Sie uns aus dem Ausland - Internationale Telefonnummern",
+                        "title": "So erreichen Sie uns - Internationale Telefonnummern",
                         "anchor": "international_phone_numbers",
                         "active": true,
                         "textblock": [
-                            "Außerhalb Deutschlands erreichen Sie uns unter folgenden internationalen Telefonnummern:",
-                            "Technische Hotline: +49 30 498 75401 erreichbar Montag bis Samstag von 7 bis 22 Uhr Ortszeit in Deutschland (außer an bundesweiten Feiertagen)",
-                            "TAN-Hotline: +49 30 498 75402 erreichbar 24 Stunden und sieben Tage die Woche",
-                            "Hinweis: Diese Rufnummern sind <b>nicht kostenfrei</b> - die Kosten für den internationalen Anruf richten sich nach dem Vertrag mit Ihrem Provider."
+                            "Sie können uns per Telefon von in und außerhalb Deutschlands erreichen. Die nationalen Telefonnummern sind für Sie kostenfrei, können jedoch nur aus dem deutschen Telefonnetz erreicht werden. Die internationalen Rufnummern sind <b>nicht kostenfrei</b> - die Kosten für den internationalen Anruf richten sich nach dem Vertrag mit Ihrem Provider.",

```

Comment:
Maybe 
"von in und außerhalb" =>
"von inner- und außerhalb" ?

Reply:
yes good idea :)